NeoML/Python/neoml/Dnn/Loss.py

```diff
@@ -741,15 +741,66 @@ def __init__(self, input_layers, loss_weight=1.0, name=None):
 
 
 class CustomLossCalculatorBase(metaclass=ABCMeta):
-    """
+    """The base class that you should implement to calculate the custom loss function.
     """
     @abstractmethod
     def calc(self, data, labels):
-        """
+        """Calculates the custom loss function.
+        This function may use only the operations supported for autodiff:
+
+        - simple arithmetic: `+ - * /`
+        - the `neoml.AutoDiff.*` functions
+        - `neoml.Autodiff.const` for creating additional blobs filled with given values
```

Comment:
It's better to mention the fact that this method should return a blob with losses for each object in a batch (otherwise user may think that he should manually calculate average loss across whole batch)